scenarioo-client/app/manage/comparisons/createComparisonModal.controller.ts

```diff
@@ -39,6 +39,9 @@ function CreateComparisonModalController($uibModalInstance, BranchesAndBuildsSer
     vm.cancel = cancel;
     vm.createComparison = createComparison;
 
+    vm.onComparisonsPage = $routeParams['tab'] === 'comparisons';
+    vm.route = $route;
```

Comment:
I dont whink that injected services should be assigned to properties on the VM - those can be used directly from inside the controller can't they?

@danielsuter or what is your recommendation on this?

Reply:
will change this as well on my other branch